code-challenges.test.js

```diff
@@ -19,44 +19,148 @@ const numbersArray1 = [6, 7, 8, 9, 10]
 const numbersArray2 = [24, 27, 30, 33, 36]
 // Expected output: [72, 81, 90, 99, 108]
 
+//I'll write out a jest test ensuring the following 4 methods are in place
+//describe,it,expect,toEqual
+//describe will ask for a function called multBy3
+//it will include what is supposed to happen in the code
+//expect will include part of future conditional statements giving desired result
+//toEqual will have the desired result
+describe("multBy3", () =>
+  it ("multBy3 takes in an array of numbers and returns an array with all the numbers multiplied by 3", () => {
+    expect (multBy3(numbersArray1)).toEqual([18, 21, 24, 27, 30])
+  })
+  )
+//Failed ReferenceError: multBy3 is not defined
+//create function with multBy3 being the name of said function
+const multBy3 = (numbersArray1) => {
+  return numbersArray1.map(value => value * 3)
+}//Test Suites: 1 passed, 1 total
+// Tests:       1 passed, 1 total
+
```

Comment:
This looks great. The goal of a function is to create reusable code. A function should take a parameter that is neutral and then a specific value for the arguments.
There only needs to be one test and one function.